packages/core/src/hooks/datadog/metrics/esbuild.ts

```diff
@@ -42,7 +42,8 @@ export const getIndexed = (stats: EsbuildStats, context: string): EsbuildIndexed
     if (Array.isArray(stats.entrypoints)) {
         // We don't have an indexed object as entry, so we can't get an entry name from it.
         for (const entry of stats.entrypoints) {
-            const realEntry = getModulePath(entry, context);
+            const fullPath = typeof entry === 'object' ? entry.in : entry;
```

Comment:
Not blocking: I didn't check the type, but if `entry` is `null`, `typeof entry` will also say `'object'`
If you want to handle such cases, you need to do `entry && typeof entry === 'object'`

Reply:
Types say that it can't be `null`, but this is still best to have this safe guard.